deployments/common/image/common-scripts/env-update

```diff
@@ -42,7 +42,8 @@ for filepath in $*; do
             ;;
         pip)
             echo "Installing pip pkgs into env ${env} from: ${filepath}"
-            pip install --no-cache-dir --use-feature=2020-resolver -r ${filepath}
+            #pip install --no-cache-dir --use-deprecated=legacy-resolver -r ${filepath}
+	    pip install --no-cache-dir --pre -r ${filepath}
```

Comment:
I was wondering if we should really use --pre here for production since it may pull in alphas just because it can whenever we spec unpinned packages.  Post demo,  potentially we could add parameters to the script and/or env vars to enable adding params temporarily.

Reply:
Removed.